examples/next-graphql/components/helpers/VisualEditor.tsx

```diff
@@ -0,0 +1,116 @@
+"use client"
+
+import { useState , ReactNode } from "react";
+import {arrayMove, SortableContext, useSortable } from '@dnd-kit/sortable';
+import {DndContext, DragEndEvent} from '@dnd-kit/core';
+import { CSS } from '@dnd-kit/utilities';
+
+import type { VisualEditorComponentProps } from "drupal-visual-editor"
+import { openVisualEditorComponent, syncDrupalComponentOrder } from "drupal-visual-editor"
+
+interface VisualEditorComponentContainerProps extends VisualEditorComponentProps {
+  children?: React.ReactNode;
+}
+
+function VisualEditorComponentContainer(
+  { action, storage, uuid, children }: VisualEditorComponentContainerProps
+) {
+  return (
+    // eslint-disable-next-line
```

Comment:
Why do we need this?

Reply:
It was needed when we used `onClick` after changing to `onDoubleClick` is no longer required

Reply:
fixed